app/views/layouts/application.html.erb

```diff
@@ -6,6 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
   <title><%= title %></title>
   <%= stylesheet_link_tag :application, media: "all" %>
+  <%= stylesheet_link_tag :budget, media: "all" %>
```

Comment:
Instead of calling this stylesheet here, import it in `app/assets/stylesheets/application...`

Reply:
Done in sha: a2fa585